parsl/executors/taskvine/utils.py

```diff
@@ -83,3 +90,8 @@ def run_parsl_function(map_file, function_file, argument_file, result_file):
     """
     from parsl.executors.taskvine.exec_parsl_function import run
     run(map_file, function_file, argument_file, result_file)
+
+
+def load_variable_in_serverless(var_name):
```

Comment:
docstring if users are meant to be calling this to tell me what it is/why its useful